src/goGenerateTests.ts

```diff
@@ -172,7 +184,10 @@ function generateTests(conf: Config, goConfig: vscode.WorkspaceConfiguration): P
 
 				vscode.window.showInformationMessage(message);
 				outputChannel.append(message);
-				if (testsGenerated) {
+
+				const isTestFile = (conf.testFile !== undefined && conf.testFile);
```

Comment:
Javascript is fun in a way that we can skip the undefined check :) Because null or undefined values would be "falsy"
And so your if check can directly be `if (testsGenerated && conf.testFile)`